packages/babel-runtime/package.js

```diff
@@ -0,0 +1,12 @@
+Package.describe({
+  name: "babel-runtime",
+  summary: "Runtime support for output of Babel transpiler",
+  version: '0.1.0',
```

Comment:
Not really sure how this package should be versioned… other than, of course, semantically.